src/components/submission/feed-entry.vue

```diff
@@ -84,12 +84,16 @@ except according to the terms contained in the LICENSE file.
         <span class="submission-feed-entry-entity-error">{{ $t('title.entity.error') }}</span>
         <span class="entity-error-message" v-tooltip.text>{{ entry.details.problem?.problemDetails?.reason ?? entry.details.errorMessage ?? '' }}</span>
       </template>
-      <template v-else>
+      <template v-else-if="comment">
         <span class="icon-comment"></span>
         <i18n-t keypath="title.comment">
           <template #name><actor-link :actor="entry.actor"/></template>
         </i18n-t>
       </template>
+      <template v-else>
```

Comment:
add special cases for `v-else-if="action.startsWith('submission.backlog.')"` events

can take inspiration from but simplify
```
 <i18n-t :keypath="`title.updateReviewState.${reviewState}.full`">
          <template #reviewState>
            <span class="review-state" :class="reviewState">
              <span :class="reviewStateIcon(reviewState)"></span>
              <span>{{ $t(`title.updateReviewState.${reviewState}.reviewState`) }}</span>
            </span>
          </template>
          <template #name><actor-link :actor="entry.actor"/></template>
        </i18n-t>
```

```$t(`title.submissionBacklog.${action.replace('submission.backlog.',''}`) ```